sig-windows/annual-report-2023.md

```diff
@@ -4,38 +4,34 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-<!--
-   Some example items that might be worth highlighting:
-   - Major KEP advancement
-   - Important initiatives that aren't tracked via KEPs
-   - Paying down significant tech debt
-   - Governance and leadership changes
--->
+  - Windows Operational Readiness [v0.1.0](https://github.com/kubernetes-sigs/windows-operational-readiness/releases/tag/0.1.0)
+    was released
+  - [Windows support for InPlace Pod Vertical Scaling](https://github.com/kubernetes/kubernetes/pull/112599)
+  - [CRI only metrics](https://github.com/kubernetes/kubernetes/pull/116968)
+  - Windows-service-proxy was highlighted as a working example of building your own kube-proxy out of tree using KPNG
+  - Kicked off work on [Image Pull Per Runtime Class](https://github.com/kubernetes/enhancements/pull/4217/) feature
+    which spans kubelet and CRI
+  - Leadership changes
+    - Aravindh Puthiyaparambil became co-chair while Mark Rossetti also took on the mantle of tech lead
+    - Jay Vyas stepped down as tech lead and was replaced by Amim Knabben
 
 2. Are there any areas and/or subprojects that your group needs help with (e.g. fewer than 2 active OWNERS)?
 
-<!--
-   Note: This list is generated from the KEP metadata in kubernetes/enhancements repository.
-      If you find any discrepancy in the generated list here, please check the KEP metadata.
-      Please raise an issue in kubernetes/community, if the KEP metadata is correct but the generated list is incorrect.
--->
+No
 
 3. Did you have community-wide updates in 2023 (e.g. KubeCon talks)?
 
-<!--
-  Examples include links to email, slides, or recordings.
--->
+  - [What's new with SIG Windows, Kubecon EU](https://sched.co/1HyTs)
+  - [What's new with SIG Windows, Kubecon NA](https://sched.co/1R2mL)
 
 4. KEP work in 2023 (v1.27, v1.28, v1.29):
 
   - Alpha
     - [2258 - Node log query](https://github.com/kubernetes/enhancements/tree/master/keps/sig-windows/2258-node-log-query) - v1.27
-
-
+    - [4216: Image pull per runtime class](https://github.com/kubernetes/enhancements/pull/4217/)
 
 ## [Subprojects](https://git.k8s.io/community/sig-windows#subprojects)
 
-
 **New in 2023:**
   - [windows-service-proxy](https://git.k8s.io/community/<no value>#windows-service-proxy)
 **Continuing:**
```

Comment:
We might need a blank line before **Continuing**
The formatting does not look correct in the formatted view

Reply:
Fixed.